webgpu.h

```diff
@@ -344,6 +344,7 @@ typedef enum WGPUSType {
     WGPUSType_ShaderModuleSPIRVDescriptor = 0x00000005,
     WGPUSType_ShaderModuleWGSLDescriptor = 0x00000006,
     WGPUSType_PrimitiveDepthClipControl = 0x00000007,
+    WGPUSType_SurfaceDescriptorFromWayland = 0x00000008,
```

Comment:
nit: Maybe `FromWaylandSurface`? I wonder if it should be `FromXlibWindow` as well instead of `FromXlib`.

@kvark WDYT?

Reply:
current seems fine. Technically Vulkan would call it `WaylandSurfaceDescriptor`, etc, but personally I like the uniformity of putting the variations of this structure at the end ("...FromXxx") instead of prefixing them.

Reply:
Sorry, what I meant was "WGPUSType_SurfaceDescriptorFromWaylandSurface" instead of "WGPUSType_SurfaceDescriptorFromWayland". We are making a `WGPUSurface` from a wayland surface, not from wayland itself. It is a very nitpicky comment though.

Reply:
to me, `SurfaceDescriptorFromWaylandSurface` does indeed make more sense, as it aligns better with other SufaceDescriptor structs (`..FromWindowsHWND`, `..FromMetalLayer`, `..FromCanvasHTMLSelector`).

Though `SurfaceDescriptorFromXlib` does seem like an outlier (which is what I looked for current `..FromWayland`).

so anyways, should I change it to `SurfaceDescriptorFromWaylandSurface` ?

Reply:
Yeah I'll make a fix for `SurfaceDescriptorFromXlib` after this. Let's use `SurfaceDescriptorFromWaylandSurface` thanks!

Reply:
updated